graylog2-web-interface/src/components/maps/configurations/GeoIpResolverConfig.tsx

```diff
@@ -175,6 +180,18 @@ const GeoIpResolverConfig = ({ config = defaultConfig, updateConfig }: Props) =>
                                disabled={!values.enabled}
                                label="Path to the ASN database"
                                name="asn_db_path" />
+                  <TimeUnitInput label="Database refresh interval"
+                                 update={(value, unit) => {
+                                   setFieldValue('refresh_interval', value);
+                                   setFieldValue('refresh_interval_unit', unit);
+                                 }}
+                                 help="Interval at which the database files are checked for modifications and refreshed changes are detected on disk."
+                                 value={values.refresh_interval}
+                                 unit={values.refresh_interval_unit || 'MINUTES'}
+                                 defaultEnabled={values.enabled}
+                                 enabled={values.enabled}
+                                 hideCheckbox
+                                 units={['SECONDS', 'MINUTES', 'HOURS', 'DAYS']} />
```

Comment:
I think it would also be helpful to display the `Database refresh interval` field on the main config page, so folks can see if it is set from the main Config page (without editing it).

![image](https://user-images.githubusercontent.com/3423655/180829722-3d9e0e60-5670-4e22-a173-4f6882f0b74a.png)

https://github.com/Graylog2/graylog2-server/blob/00371b4a4a3d3f84c5241eb0ec78a873dd0e60a3/graylog2-web-interface/src/components/maps/configurations/GeoIpResolverConfig.tsx#L108-L115

Reply:
Super nitpicky and not important to fix for this particular PR, but with the addition of the longer config names the spacing in this plugin config UI seems to have strayed from the typical inline formatting of the other plugins. @mikedklein maybe something frontend folks could clean up when there's time.
![image](https://user-images.githubusercontent.com/91903901/181027841-4a7d333c-4cf9-4f91-8d3d-1bb13f0ecc42.png)